rest/blip_client_test.go

```diff
@@ -336,16 +354,26 @@ func (btc *BlipTesterClient) getLastReplicatedRev(docID string) (revID string, o
 	return revID, ok
 }
 
-func newBlipTesterReplication(id string, btc *BlipTesterClient) (*BlipTesterReplicator, error) {
+func newBlipTesterReplication(id string, isPull bool, btc *BlipTesterClient) (*BlipTesterReplicator, error) {
```

Comment:
How about splitting into two functions for ease of use/clarity - newBlipTesterPushReplication and newBlipTesterPullReplication?